3rd/wepoll/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.11)
+project(wepoll)
+include(FetchContent)
+FetchContent_Declare(
```

Comment:
I suggest putting the wepoll source code directly into event/wepoll/ directory, similar to event/kcp/, because not all regions can access github smoothly.